lib/upipe-modules/upipe_udp.c

```diff
@@ -572,7 +572,7 @@ int upipe_udp_open_socket(struct upipe *upipe, const char *_uri, int ttl,
         else if (bind_addr.ss.ss_family != AF_UNSPEC) {
     normal_bind:
             if (bind(fd, &bind_addr.so, sockaddr_len) < 0) {
-                upipe_err(upipe, "couldn't bind");
+                upipe_err(upipe, "couldn't bind: %m");
```

Comment:
Does it work ? I would think you need upipe_err_va for that to work. Also I thought %m was non-portable GNU extension.